services/QuillLMS/app/workers/populate_annual_vitally_worker.rb

```diff
@@ -0,0 +1,28 @@
+class PopulateAnnualVitallyWorker
+  include Sidekiq::Worker
+
+  USER_ROLES_TO_SYNC = ['teacher', 'admin', 'auditor']
+
+  def perform
+    # Don't synchronize non-production data
+    return unless ENV['SYNC_TO_VITALLY'] == 'true'
+
+    year_to_sync = Date.today.year - 1
+    schools_to_sync.each_slice(100) do |school_batch|
+      school_ids = school_batch.map { |school| school.id }
+      SyncVitallyPastYearAccountsWorker.perform_async(school_ids, year_to_sync)
+    end
+    users_to_sync.each_slice(100) do |user_batch|
+      user_ids = user_batch.map { |user| user.id }
+      SyncVitallyPastYearUsersWorker.perform_async(user_ids, year_to_sync)
```

Comment:
You can use [pretzel colon](https://technology.customink.com/blog/2015/06/08/ruby-pretzels/) here:  `user_ids = user_batch.map(&:id)`